### ISSUE TITLE
1D distributions of 2D histograms with colorZvar

### ISSUE THREAD

**Issue by ehellbar** [miranov25/RootInteractive#147]
Hello @pl0xz0rz  and @miranov25 ,

we were also thinking of drawing the 1D distributions of 2(or more)D histograms with markers, the same way 1D histograms are drawn, but also having the second (or a given) dimension on the z color scale. 

So basically instead of
x -> var0
y -> var1
z -> #entries

we would like to draw
x -> var1
y -> #entries
z -> var0

Is it already possible? If not, I think it could be a nice feature.

Thanks again,
Ernst

**Comment by ehellbar** [comment]
Hello @miranov25 ,
here I refer to a different feature than what was addressed in #148 .

**Comment by ehellbar** [comment]
These definition produces exactly the same plot. Thanks @pl0xz0rz .
```
histoArray = [
    {"name": "hisdiff3D", "variables": ["r", "diffCorrR"],"nbins": [5, 75], "axis": [0,1]},
]

figureArray = [
    [['hisdiff3D.bin_center_1'], ['hisdiff3D.bin_count'], {'colorZvar': 'hisdiff3D.bin_center_0', 'size': 3}],
]
```
